api/api.proto

```diff
@@ -381,6 +385,14 @@ message PXDProxySpec {
     string id = 1;
 }
 
+// PureBlockSpec is the spec for proxying a volume on pure_block backends
+message PureBlockSpec {
```

Comment:
Do we want to add some sort of "TODO" here for implementing this later, or specifically leave it out? @ggriffiths @pure-jliao 

I can think of reasons for both so I'm generally indifferent, just want to propose the idea

Reply:
I created this just for future usage, now FA pass through is not supported and requests will be rejected if the backend is pure_block